tsconfig.json

```diff
@@ -23,6 +24,7 @@
     "strict": true,
     "strictNullChecks": true,
     "target": "es5",
+    "tsBuildInfoFile": "node_modules/.cache/.tsbuildinfo",
```

Comment:
I looked up the docs for this setting and don't understand exactly why it's being used here in this way--help?

Reply:
`tsBuildInfoFile` is a file written by `tsc` when `incremental: true`, so that it can save compiler state to disk and use it to speed up future builds. This option just specifies where that file should be written.